gitlab/data_source_gitlab_branch.go

```diff
@@ -0,0 +1,74 @@
+package gitlab
+
+import (
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
```

Comment:
Please upgrade to the [terraform SDK v2](https://www.terraform.io/plugin/sdkv2/sdkv2-intro) and make sure to rebase to an up-to-date `master` branch.